src/ui_lyrics_panel.h

```diff
@@ -24,6 +27,7 @@ class LyricPanel : public CWindowImpl<LyricPanel>, protected ui_config_callback_
     void on_playback_dynamic_info_track(const file_info& info) override;
     void on_playback_time(double /*time*/) override {}
     void on_volume_change(float /*new_volume*/) override {}
+    const LyricData& get_lyrics() const { return m_lyrics; } // m_lyrics getter

```

Comment:
This comment doesn't add any value.

```suggestion
    const LyricData& get_lyrics() const { return m_lyrics; }
```